backend/pkg/logger/data/logger.go

```diff
@@ -143,7 +165,13 @@ func (sublogger *Logger) Stop() error {
 	}
 
 	for _, file := range sublogger.valueFileSlice {
-		file.Close()
+		err := file.Close()
+		if err != nil {
+			return loggerHandler.ErrClosingFile{
+				Name:      Name,
+				Timestamp: time.Now(),
+			}
+		}
```

Comment:
This will stop closing the files on the first error and might leave some files opened. Instead just print the errors or store them for later use and first run `file.Close()` for all files